apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -311,6 +317,24 @@ export class ModalPage {
     await signButton.click()
   }
 
+  async signTypedData() {
+    const signButton = this.page.getByTestId('sign-typed-data-button')
+    await signButton.scrollIntoViewIfNeeded()
+    await signButton.click()
+  }
+
+  async signMessageAndTypedData(modalValidator: ModalValidator, network?: string) {
+    await this.sign()
+    await modalValidator.expectAcceptedSign()
+
+    if (network !== 'Solana') {
```

Comment:
I'd go with network === 'eip155' even though we don't yet have btc in the extension (it's more future proof)